### DISCUSSION TITLE
Two questions/comments about Jupyter

### DISCUSSION THREAD

**Discussion by aroberge** [Textualize/rich#1375] [General] [answered]
In the documentation, you mention that one should do
```
pip install rich[jupyter]
```
to install additional dependencies when using Rich with Jupyter.  I had never bothered to do so (since everything worked just by installing Rich). I just tried to run that command and it didn't install anything additional.  Is there something I am missing?  (Perhaps this is no longer needed and the documentation should be updated.)

When using Rich with Jupyter, you specify a series of font-families in https://github.com/willmcgugan/rich/blob/master/rich/jupyter.py#L8, starting with `Menlo` and ending with `monospace`.  I have found that, at least on Windows, using Rich results in seemingly larger font size than regular Jupyter output. Is there any reason for specifying this font-family style?

(In my project I decided to monkeypatch `JUPYTER_HTML_FORMAT` to remove the font-family choice so as to make the output the same as the Jupyter's default ... which prompted me to post this.)

**Comment by willmcgugan** [comment, marked as answer]
The Jupyter extras installs `ipywidgets` which I'm guessing you already had. Perhaps Jupyter has that as a dependency now. Let me know if it doesn't need it and I'll update the docs.

The fonts were chosen to work well across the various combinations of OS and browser. Some fonts don't produce good output with the box characters, they may not align or you see gaps in the rows. The existing choices were a compromise. You may find that you can improve it on one platform only by breaking another.

**Reply by aroberge** [reply]
Thanks.

I did a new clean install and indeed ipywidgets were not installed by default -- my mistake.  However I had never bothered to install the rich jupyter extension in my development environment and everything I wanted to do with rich **just worked** in Jupyter notebooks and JupyterLab  (and Google Colab).
